stock_rule_reserve_max_quantity/__manifest__.py

```diff
@@ -3,8 +3,9 @@
 
 {
     "name": "Stock Rule Reserve Max Quantity",
-    "summary": "Allows to reserve max available quantity when a move comes from an stock rule",
-    "version": "16.0.1.0.1",
+    "summary": "Allows to reserve max available quantity "
+    "when a move comes from an stock rule",
```

Comment:
suggestion: If you surround split strings with parenthesis, ruff will make it prettier.